syncer/types.go

```diff
@@ -70,6 +70,8 @@ type Syncer interface {
 	HasSyncPeer() bool
 	// Sync starts routine to sync blocks
 	Sync(func(*types.FullBlock) bool) error
+	// Indicates if syncer is syncing with the best peer
```

Comment:
Nitpick, consistency-wise:
```suggestion
	// IsSyncing indicates if syncer is syncing with the best peer
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/2020/commits/2efd42c0766ca2e4e0eea71f93388185086c5134